index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="white-background">
+      <header>
+        <a class="logo">my logo</a>
```

Comment:
[OPTIONAL] Great job changing the the tag from div to a. But if you want it to clickable please add href value. So it will look like `<a href="#">`.